src/pages/RepoPage/FailedTestsTab/FailedTestsPage/FailedTestsPage.tsx

```diff
@@ -1,128 +1,12 @@
-import { useParams } from 'react-router-dom'
-
-import {
-  TIME_OPTION_VALUES,
-  TimeOption,
-  TimeOptions,
-} from 'pages/RepoPage/shared/constants'
-import { useLocationParams } from 'services/navigation'
-import { useRepoOverview } from 'services/repo'
-import A from 'ui/A'
-import MultiSelect from 'ui/MultiSelect'
-import Select from 'ui/Select'
-
-import BranchSelector from './BranchSelector'
 import FailedTestsTable from './FailedTestsTable'
 import { MetricsSection } from './MetricsSection'
-
-interface URLParams {
-  provider: string
-  owner: string
-  repo: string
-  branch?: string
-}
-
-const getDecodedBranch = (branch?: string) =>
-  !!branch ? decodeURIComponent(branch) : undefined
+import SelectorSection from './SelectorSection/SelectorSection'
```

Comment:
nit: it would be cleaner if you create index.ts and import this from there

Reply:
Fixed in [cd42e8b](https://github.com/codecov/gazebo/pull/3231/commits/cd42e8b26fda1c96c32b70431db7d886d03ff06c)